mllib/src/test/scala/org/apache/spark/ml/feature/NGramSuite.scala

```diff
@@ -84,7 +84,7 @@ class NGramSuite extends MLTest with DefaultReadWriteTest {
 
   def testNGram(t: NGram, dataFrame: DataFrame): Unit = {
     testTransformer[(Seq[String], Seq[String])](dataFrame, t, "nGrams", "wantedNGrams") {
-      case Row(actualNGrams : Seq[String], wantedNGrams: Seq[String]) =>
+      case Row(actualNGrams : Seq[_], wantedNGrams: Seq[_]) =>
```

Comment:
Just curious, why change `Seq[String]` to `Seq[_]`, is it for fixing some potential issue ?

Reply:
String is not actually checked because of erasure, so IntelliJ complained with a style warning before this change.